cypress/e2e/pages/testsForAllAMPPages.js

```diff
@@ -23,21 +23,32 @@ export const testsThatFollowSmokeTestConfigForAllAMPPages = ({
         // live radio pages and error pages do not have an image, media asset pages do not have the most watched list with images on amp
         if (!pageTypesNoImages.includes(pageType)) {
           it('should have webp images on pages', () => {
-            // on amp there are hidden embed images, so we check only ichef ones
-            cy.get('amp-img[src*="ichef."], img[src*="ichef."]').each($img => {
-              // when you use a .each loop or other JS function that take a callback function (here with $img that is executed for each image element)
-              // you leave the Cypress command queue and are using plain JS. Using .wrap converts
-              // the JQuery element into a Cypress wrapped element so we can execute Cypress commands on it
+            cy.document().then(doc => {
```

Comment:
Can this be moved to testsForAllPages, to prevent duplication between amp & canonical code? (Is there a reason why the tests are different, if you're checking for the existence of both an amp-img and/or an img?)

Reply:
You are right, I added in the checking for more than one type (img or amp-img) after differentiating them, but this way I can put them back together!

Reply:
I have just remembered that I actually deleted testsForAmpOnly as we don't support amp on home pages officially anymore. Maybe we actually need to discuss this though so I'll put it back for now.